plugins/dynamix.docker.manager/include/CreateDocker.php

```diff
@@ -593,8 +593,12 @@ function setXmlVal(&$xml, $value, $el, $attr = null, $pos = 0) {
         foreach ($xml['Config'] as &$arrConfig) {
           if ($arrConfig['Type'] == 'Path' && strtolower($arrConfig['Target']) == '/config') {
             $arrConfig['Default'] = $arrConfig['Value'] = realpath($dockercfg["DOCKER_APP_CONFIG_PATH"]).'/'.$xml["Name"];
-            $arrConfig['Display'] = 'hidden';
-            $arrConfig['Name'] = 'AppData Config Path';
+            if (empty($arrConfig['Display'])) {
+              $arrConfig['Display'] = 'advanced';
+            }
+            if (empty($arrConfig['Name'])) {
```

Comment:
For V1 templates, `$arrConfig['Name']` will always be something like 'Path 1'.  If we check for empty-ness here the Name will never be set to 'AppData Config Path'.

Same for Display, it'll always be 'always'.

I guess the purpose of this was to hide the /config path so it's one less thing for the user to fill out since unRAID should be installing (and managing) it off of their appdata share.